tools/labs/Makefile

```diff
@@ -39,6 +39,12 @@ docs:
 	cp -r $(KDIR)/Documentation/output/slides/teaching/_static $(KDIR)/Documentation/output/teaching/
 
 clean::
-	-rm -rf skels
+	find skels -type f \( -name *.mod.* -or -name *.ko -or -executable -or -name *.cmd -or -name *.o \) | xargs --no-run-if-empty rm || true
+	find skels -type f \( -name modules.order \) | xargs --no-run-if-empty rm || true
+	rm skels/Module.symvers || true
+	rm -r skels/.tmp_versions
 
-.PHONY: skels build copy docs clean
+clean_skels:
+	rm -rf skels
```

Comment:
Add the `-` (minus) prefix here as well.

Reply:
We should do this in a later PR.